proto/rill/admin/v1/api.proto

```diff
@@ -690,6 +690,31 @@ service AdminService {
   rpc ListPublicBillingPlans(ListPublicBillingPlansRequest) returns (ListPublicBillingPlansResponse) {
     option (google.api.http) = {get: "/v1/billing/plans"};
   }
+
+  rpc RequestProjectAccess(RequestProjectAccessRequest) returns (RequestProjectAccessResponse) {
+    option (google.api.http) = {
+      post: "/v1/organizations/{organization}/projects/{project}/access-request",
+      body: "*"
+    };
+  }
+
+  rpc GetProjectAccess(GetProjectAccessRequest) returns (GetProjectAccessResponse) {
+    option (google.api.http) = {get: "/v1/organizations/{organization}/projects/{project}/access-request/{id}"};
+  }
```

Comment:
I know it stutters, but I think this needs to be called `GetProjectAccessRequest(GetProjectAccessRequestRequest)` because right now it sounds like the API returns info about a given user's access permissions.